### PR TITLE
[Fix] Remove the inplace operation in  uper_head and fpn_neck

### PR DESCRIPTION
**Pull request by RockeyCoss**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Remove the inplace operation in uper_head，which may cause RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation.

## Modification

Remove the inplace operation in uper_head，which may cause RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation.

### CODE REVIEW

**Comment by Junjun2016** [comment]
Hi @RockeyCoss 
Could add more details about the error?
And which specific case will cause this error?

**Comment by RockeyCoss** [comment]
> 

vit+mln+upernet raises RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation.
图片
图片

swin + upernet doesn't raise Error.
resnet+upernet doesn't raise Error.

**Comment by RockeyCoss** [comment]
from pytorch's documentation:
图片

**Comment by hhaAndroid** [comment]
@RockeyCoss Will the same problem occurs in single GPU training?

**Comment by Junjun2016** [comment]
The CI failed.

File: mmseg/models/necks/fpn.py [hunk 1]
```diff
@@ -175,10 +175,11 @@ def forward(self, inputs):
             # In some cases, fixing `scale factor` (e.g. 2) is preferred, but
             #  it cannot co-exist with `size` in `F.interpolate`.
             if 'scale_factor' in self.upsample_cfg:
-                laterals[i - 1] += resize(laterals[i], **self.upsample_cfg)
+                laterals[i - 1] = laterals[i - 1] + resize(
+                    laterals[i], **self.upsample_cfg)
```
Comment (Junjun2016) [inline mmseg/models/necks/fpn.py:179]: Improve unittest coverage (missed line).

**Review by Junjun2016 — APPROVED** [review]
LGTM

**Pull request merged**